gulpfile.babel.js

```diff
@@ -14,6 +17,31 @@ import sassGlob from 'gulp-sass-glob';
 import sourcemaps from 'gulp-sourcemaps';
 import stylelint from 'stylelint';
 import syntaxScss from 'postcss-scss';
+import webpack from 'webpack';
+
+// TODO: work out how to inject this file's config object properties for js into the webpack config
+import webpackConfig from './webpack.config.babel.js';
+
+function handleJsBuild(done) {
+  return (err, stats) => {
+    if (err) {
+      log('Error', err);
+      if (done) {
+        done();
```

Comment:
Will look into if this has to throw, or do something else, in order to halt the pipeline if the js build fails.